packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -45,5 +48,27 @@ export abstract class TableColumn<
             );
         }
         this.columnInternals = new ColumnInternals(options);
+        this.columnInternals.currentSortDirection = this.sortDirection;
+        this.columnInternals.currentSortIndex = this.sortIndex;
+    }
+
+    protected sortDirectionChanged(): void {
+        this.columnInternals.currentSortDirection = this.sortDirection;
+    }
+
+    protected sortIndexChanged(): void {
+        this.columnInternals.currentSortIndex = this.sortIndex;
+    }
+
+    protected sortingDisabledChanged(): void {
+        if (this.columnInternals) {
```

Comment:
Why do we need the check here for `this.columnInternals` being defined but not in `sortDirectionChanged` or `sortIndexChanged` above? I'm guessing because `sortDirection` and `sortIndex` are initialized to the default value for their types so the change handlers aren't called during their initialization?

Reply:
Correct. The FAST `setValue` function is called for the initialization of all 3 properties when a column is created. But for sortDirection and sortIndex, `oldValue` and `newValue` are both `undefined` so the `Changed()` function isn't called inside `setValue`.

Reply:
Can you include a comment? It was pretty unintuitive for me why the check is here.

Something like "ignores the default value sortingDisabled initialization from undefined to false as it runs before columnInternals finishes initializing"

Reply:
Updated